### PR TITLE
Add BlobToolkit to the list of interactive tools

### PR DESCRIPTION
**Pull request by Delphine-L**
Hi, I'd like to get the Interactive BlobToolkit to the list of interactive tools and get in on main. Let me know if there is a better way to do that or if I am missing a step. 

Tested on eu and it seems to be working fine. 

Source : https://github.com/usegalaxy-eu/galaxy/blob/release_24.1_europe/tools/interactive/interactivetool_blobtoolkit.xml

## How to test the changes?
- [x] Instructions for manual testing are as follows:
  1. Verify the tool is installed and working by following the GTN material [ERGA post-assembly QC](https://training.galaxyproject.org/training-material/topics/assembly/tutorials/ERGA-post-assembly-QC/tutorial.html) for example

## License
- [x] I agree to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT).

### CODE REVIEW

File: tools/interactive/interactivetool_blobtoolkit.xml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<tool id="interactive_tool_blobtoolkit" tool_type="interactive" name="Interactive BlobToolKit" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="21.05">
+    <description>genome assembly QC viewer</description>
+    <macros>
+        <token name="@TOOL_VERSION@">4.1.0</token>
+        <token name="@VERSION_SUFFIX@">1</token>
+    </macros>
+    <requirements>
+        <container type="docker">quay.io/galaxy/blobtoolkit-server:4.1.0</container>
```
Comment (bgruening) [inline tools/interactive/interactivetool_blobtoolkit.xml]:
```suggestion
        <container type="docker">quay.io/galaxy/blobtoolkit-server:@TOOL_VERSION@</container>
```

File: tools/interactive/interactivetool_blobtoolkit.xml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+<tool id="interactive_tool_blobtoolkit" tool_type="interactive" name="Interactive BlobToolKit" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="21.05">
+    <description>genome assembly QC viewer</description>
+    <macros>
+        <token name="@TOOL_VERSION@">4.1.0</token>
+        <token name="@VERSION_SUFFIX@">1</token>
```
Comment (bgruening) [inline tools/interactive/interactivetool_blobtoolkit.xml]:
```suggestion
        <token name="@VERSION_SUFFIX@">0</token>
```

**Comment by Delphine-L** [comment]
Could we install it without waiting for 24.2 when it's ready ?

File: tools/interactive/interactivetool_blobtoolkit.xml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        <entry_point name="Blobtoolkit View" requires_domain="True">
+            <port>80</port>
+            <url>view/all</url>
+        </entry_point>
+    </entry_points>
+    <command detect_errors="exit_code"><![CDATA[
+    ## The following is a hack. Our embedded pulsar is not respecting the Galaxy jwd-tmp dir.
+    export TEMP=\$PWD &&
+    export TMP=\$PWD &&
+    export TMPDIR=\$PWD &&
```
Comment (mvdbeek) [inline tools/interactive/interactivetool_blobtoolkit.xml:20]: Is this some old copy and paste ? If there is a problem with embeded pulsar and the temp dir handling issue should be opened.

**Review by mvdbeek — APPROVED** [review]

File: tools/interactive/interactivetool_blobtoolkit.xml (hunk 3 above)
Comment (mvdbeek) [reply tools/interactive/interactivetool_blobtoolkit.xml:20]: @bgruening @kysrpex do you know where this originated from, is this still needed ?

**Comment by cat-bro** [comment]
Thanks for adding this! A couple of Galaxy Australia users have been asking for it. Can this go onto the current release branch?

**Pull request merged**